web/src/main/webapp/resources/js/custom.js

```diff
@@ -14,6 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+var linePlot;
```

Comment:
We should namespace our javascript code rather than adding it as globals. I'll pull together some references on this for later work.